flow-minor-mode.el

```diff
@@ -138,6 +138,18 @@ BODY progn"
     (compilation-mode)
     (setf buffer-read-only t)))
 
+(defun flow-minor-colorize-buffer ()
+  (if (require 'web-mode nil 'noerror)
```

Comment:
I'm not sure if it is a good idea to call require directly, because if web-mode is not in the cache this can be a potentially slow operation. Another more important issue is still an implicit dependency on a major mode, i'm suppose that is not how emacs packages should be organized. So i guess we have to wait for someone to implement this feature, or borrow this somewhere else, like from a typescript mode.